icu4c/source/test/cintltst/cintltst.vcxproj

```diff
@@ -75,7 +75,6 @@
       <AdditionalLibraryDirectories>..\..\..\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <ProgramDatabaseFile>.\x86\Debug/cintltst.pdb</ProgramDatabaseFile>
       <SubSystem>Console</SubSystem>
-      <RandomizedBaseAddress>false</RandomizedBaseAddress>
       <FixedBaseAddress>false</FixedBaseAddress>
```

Comment:
I see this in a couple of files. Is it unnecessary/redundant now? I suggest doing a grep for "BaseAddress" in _all_ files.

Reply:
Right, this is redundant/unnecessary now. (This sets `/FIXED:NO` when linking, which is also the default, so its unneeded in the `vcxproj` file).
I'll grep for "BaseAddress" in all files and remove any remaining left-overs.
Thanks Markus!